### ISSUE TITLE
S3 Prefix(folder) Delete recursively

### ISSUE THREAD

**Issue by mbstacy** [culibraries/cloud-browser-django-app#4]
S3 Prefix(folder) Delete recursively

**Comment by mbstacy** [comment]
Permissions are not working correctly. With 'rw' perms the delete does not work.

**Comment by mbstacy** [comment]
@sabetiv Can you add the "bname" attribute with your post data? Permission denied because "bname" not given!

**Comment by sabetiv** [comment]
Yes.  I can send the bname in the delete post data.  There is an assumption that all prefixes and keys in the post data will belong to the same bucket.

**Comment by mbstacy** [comment]
@sabetiv yes, all prefixes and keys in the post data will belong to the same bucket.
